docs/release-notes.md

```diff
@@ -1,150 +1,149 @@
-# MCUboot release notes
+# Release notes
 
 - Table of Contents
 {:toc}
 
 ## Version 1.8.0
 
-The 1.8.0 release of MCUboot contains numerous fixes, and adds support
-for the NuttX RTOS, and the Espressif ESP32 SDK.
+The 1.8.0 release of MCUboot contains numerous fixes and adds support
+for the NuttX RTOS and the Espressif ESP32 SDK.
 
 ### About this release
 
-- Add support for the NuttX RTOS.
-- Add support for the Espressif ESP32 SDK.
+- Added support for the NuttX RTOS.
```

Comment:
I'm going to leave a bit -1 in this grammar change. The use of the present tense is very intentional, and overall makes the release notes much easier to read. The same goes for commit text, which should also be written in the present tense.

Reply:
@d3zd3z  While I personally prefer the past tense, I don't have any issue with switching them all to the present tense, as long as they are consistent if that's the issue. I just think that the current mix of tenses does not make much sense.

Reply:
Actually, the discussion is not between *present* or *past* tense, because that would assume that the verb goes in the Indicative mood, which should not be the case.
A Release Notes (as well as Git commit messages) usually contains verbs in the Imperative mood, which are more suitable to point that *things will change from this point forward*.

Reply:
Kind of drifting off topic, but there are languages that have a past imperative tense.

More seriously, do you think we should add comments to the release notes process docs stating the release notes should be in imperative mood? I see that a lot more mixed that the text of git commits, but I find it easier to go from git commits in imperative mood to release notes based on those.

BTW, I have seen the indicative present in commit text. It feels weird to me. "Adding support for ...", although I'm not sure if that is actually present indicative "I am adding support".

Reply:
@gustavonihei The imperative mood is used usually as a convention for the first line of commit messages because also commands like git merge or git revert automatically create messages in imperative. I guess one could use the argument that it makes sense to give "orders" to the codebase in imperative mood.

But we are talking about release notes here. Do you really want your release notes to be just a copy-paste of commit messages? In the context of a release note, it doesn't really make much sense to me. When the release note is published, the "order" to the codebase is something that happened in the past, so using the imperative mood in this context looks plain wrong to me. To whom are you ordering the verb's action? To the reader? It doesn't make sense to me.

But, assuming this does actually makes sense, it doesn't look like the release notes are actually consistent in following your own rule.

Anyway, even if I think this is plain wrong, I'm just an external contributor here, so if you guys think my edit of the tenses here is "controversial", I can revert it and keep the rest of the changes.